scripts/test_e2e.sh

```diff
@@ -1,69 +1,10 @@
 #!/bin/bash
 
-# test_e2e.sh
-# Execute end-to-end (e2e) tests to verify that everything is working right
-# from the end user perpsective
-set -xeuo pipefail
```

Comment:
@arschles what does `set -xeuo pipefail` exactly do? I assume it has something to do with exit codes...if you look at this script, it should exit with the proper code if `go build` fails to download the modules no?

Reply:
@arschles would still like to be educated on the command above, but in any case i moved the shell out into manual steps just so that we can have both travis && drone ci at the same time. This way we can gracefully migrate if we decided to go with drone

Reply:
If you haven't got an answer already, from [bash builtin set command](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html):
* -x "Print a trace of simple commands, for commands, case commands, select commands [...] after they are expanded and before they are executed
* -e "Exit immediately" when a command fails
* -u "Threat unset variables and parameters other than the special parameters '@' or '*' as an error when performing parameter expansion."
* -o pipefile "If set, the return value of a pipeline is the value of the last command to exit with a non-zero status, or zero if all commands in the pipeline exist successfully."